[DBS-004] Patch for Zabbix v4.4.0 to extend popup menu of problems page
--- zabbix-4.4.5-orig/frontends/php/assets/styles/hc-light.css	2019-10-24 17:03:59.655264681 +0300
+++ zabbix-4.4.5/frontends/php/assets/styles/hc-light.css	2019-10-24 17:06:36.262065976 +0300
@@ -2955,7 +2955,7 @@
   z-index: 1000;
   padding: 5px 15px 5px 25px;
   min-width: 166px;
-  max-width: 250px;
+  max-width: 450px;
   background-color: #fff;
   border: 1px solid #383838;
   border-bottom-color: #353535;
--- zabbix-4.4.5-orig/frontends/php/assets/styles/blue-theme.css	2019-10-24 17:03:20.982560653 +0300
+++ zabbix-4.4.5/frontends/php/assets/styles/blue-theme.css	2019-10-24 17:05:57.777360569 +0300
@@ -2970,7 +2970,7 @@
   z-index: 1000;
   padding: 5px 15px 5px 25px;
   min-width: 166px;
-  max-width: 250px;
+  max-width: 450px;
   background-color: #fff;
   border: 1px solid #ccd5d9;
   border-bottom-color: #c9d3d7;
--- zabbix-4.4.5-orig/frontends/php/assets/styles/dark-theme.css	2019-10-24 17:03:31.517480028 +0300
+++ zabbix-4.4.5/frontends/php/assets/styles/dark-theme.css	2019-10-24 17:06:11.212257729 +0300
@@ -2994,7 +2994,7 @@
   z-index: 1000;
   padding: 5px 15px 5px 25px;
   min-width: 166px;
-  max-width: 250px;
+  max-width: 450px;
   background-color: #2b2b2b;
   border: 1px solid #383838;
   border-bottom-color: #353535;
--- zabbix-4.4.5-orig/frontends/php/assets/styles/hc-dark.css	2019-10-24 17:03:51.022330752 +0300
+++ zabbix-4.4.5/frontends/php/assets/styles/hc-dark.css	2019-10-24 17:06:25.699146834 +0300
@@ -2955,7 +2955,7 @@
   z-index: 1000;
   padding: 5px 15px 5px 25px;
   min-width: 166px;
-  max-width: 250px;
+  max-width: 450px;
   background-color: #000;
   border: 1px solid #666;
   border-bottom-color: #636363;
